diff --git a/komorebi/src/process_event.rs b/komorebi/src/process_event.rs
index cca26f2..298d6d8 100644
--- a/komorebi/src/process_event.rs
+++ b/komorebi/src/process_event.rs
@@ -397,8 +397,6 @@ impl WindowManager {
                             if behaviour.float_override {
                                 workspace.floating_windows_mut().push(window);
                                 workspace.set_layer(WorkspaceLayer::Floating);
-                                let mut floating_window = window;
-                                floating_window.center(&workspace.globals().work_area)?;
                                 self.update_focused_workspace(false, false)?;
                             } else {
                                 match behaviour.current_behaviour {
